test/library/draft/Shlex/Shlex.chpl

```diff
@@ -0,0 +1,443 @@
+/*
```

Comment:
Can you add an empty `Shlex.notest` file to this directory so that `start_test` doesn't assume this file is a test?